.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+Reviewers: GitHub @<name>s of all reviewers
+
+Linked PRs: <links to corresponding PRs, optional>
+
+## Changes
+- <changes here>
+
+## TODOs:
+- [ ] +1
+- [ ] Updated README
+- [ ] Updated CHANGELOG
+- [ ] <Other applicable TODOs>
+- [ ] <Feedback from reviewers>
+
+## How To Test
+- <necessary config changes>
+- <necessary corresponding PRs>
+- <how to access the new / changed functionality -- fixtures, URLs>
+
+## Applicable Research Resources
+- <links, optional>
```

Comment:
I've been trying to get PR templates standardized in [this code-style PR](https://github.com/mobify/mobify-code-style/pull/97). I've also just been adding these to my other "modernizing" plugin PRs to be consistent. IMO a rough first pass is all that's currently necessary.